### DISCUSSION TITLE
/api/v2/stats has been blocked by CORS policy: Request header field access-control-allow-origin is not allowed by Access-Control-Allow-Headers in preflight response.

### DISCUSSION THREAD

**Discussion by moyinzunza** [blockscout/blockscout#9801] [Support]
I can't set the CORS

I have this error in the frontend
![image](https://github.com/blockscout/blockscout/assets/13249070/9a225e5d-c6c8-405a-97e6-db6516bfcf19)

I ran the backend with
mix phx.server

I don't use docker I saw docker-compose/proxy/default.conf.template and I edited that file whit this
add_header 'Access-Control-Allow-Origin' '*' always;

but it did not work I supose it's cuz that file is only if you run with docker

someone can help me pls?

Sorry about my english btw it's not my native language

**Comment by GimluCom** [comment]
The nginx proxy will serve both the frontend and backend over port 80 by default. It looks like you are not using nginx on neither of them.

https://discord.gg/blockscout is the easiest way to get assistance to configure Blockscout.
